secrets.properties

```diff
@@ -0,0 +1 @@
+MAPS_API_KEY=AIzaSyDP-kCBUY6_yMJiPuKWZvaF1-uPhMFvx6M 
```

Comment:
This file will be removed soon and this file will be added to the .gitignore. Make sure you have a backup of this somewhere.